server/lib/enrich/article.js

```diff
@@ -31,10 +31,17 @@ module.exports = exports = function article(content, format) {
 		content.bodyHTML = content.body;
 	}
 
+	if (content.contentStats) {
+		content.wordCount = content.contentStats && content.contentStats.wordCount;
+
+		content.hasGraphics = Boolean(content.contentStats && content.contentStats.graphics);
```

Comment:
This would clash with https://github.com/Financial-Times/next-syndication-api/pull/235/files#diff-910a4af1fe15e5abac33ad89fdc153a1R34

Reply:
True! I can create a small PR to add this property to master and we both rebase our branch.

Reply:
I think either way I'll need to rebase 😅